scenes/LevelExit.tscn

```diff
@@ -15,6 +15,7 @@ gravity_vec = Vector2( 0, 1 )
 gravity = 98.0
 linear_damp = 0.1
 angular_damp = 1.0
+collision_layer = 4
```

Comment:
Not sure, but shouldn't things like doors have their own collision layer? E.G. so that the player can enter them and the enemies just pass through?

Reply:
That's exactly how it's set in case of LevelExit though. It is on Terrain layer and its mask is Player. It doesn't care about Enemies. What benefit would it have to set it separately?

Reply:
Maybe I am misunderstanding something, but if LevelExit is on terrain layer (doesn't matter what it cares about in this case), and monsters are on enemy layer and care about terrain, wouldn't the collision of a monster and the LevelExit still be reported (both in the monster node and the level exit node)? I've seen somewhere on the  internet that the relation is bidirectional, i.e. only one has to care about the other for the collision to be reported.

Ok, found: https://godotengine.org/qa/3020/collision-masks-and-its-propper-uses

If this is correct, I think we will need another layer for doors, But I'll first try and check it in the editor.

Reply:
Yep. Just tested. Try moving the level exit down in Level2